helpers/challenge_helper.go

```diff
@@ -0,0 +1,46 @@
+package helpers
+
+import (
+	"github.com/drift-org/backend/models"
+	"github.com/kamva/mgm/v3"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+func milesToMeter(miles uint8) float64 {
+	return float64(miles) * 1609.344
```

Comment:
nit: Promote `1609.344` to constant

Reply:
This value is only used once though. Also, I don't think the miles to meter conversion rate is changing any sooner.

Reply:
Even though its may only be used once, its still common practice to separate out numbers like these to constants. For readability, nobody knows what `1609.344` is, but if you have `const METERS_PER_MILE = 1609.344`, it becomes self documenting.